docs/internals/contributing/writing-documentation.txt

```diff
@@ -321,6 +321,44 @@ documentation:
 * To improve readability, use ``.. admonition:: Descriptive title`` rather than
   ``.. note::``. Use these boxes sparingly.
 
+Directive Content Indentation
```

Comment:
You've added a header within a bullet section and so the `* Use these heading styles:` is now within the indentation section
I would either make these all bullets and remove the header or this needs to go at the bottom of the section